Monika After Story/game/script-topics.rpy

```diff
@@ -4940,3 +4940,297 @@ label monika_asks_family:
                 m 1f "You can tell me about your family when you're ready, [player]." 
                 m 1j "I love you very much!"                              
     return
+    
+
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_fun_facts_open",category=['misc'],prompt="Fun facts",pool=True))
+
+default fun_facts_start = 0
+default persistent._mas_funfactfun = True
+
+label monika_fun_facts_open:
+    #Determines the fact number and whether it's a bad fact
+    python:
+        fun_facts_bad_chance = random.randint(1,100)
+
+    #If player has opened fun facts this session
+    if fun_facts_start == 1:
+        m 1b "Up for another fun fact [player]?"
+        if persistent._mas_funfactfun == True:
+           m 2j "That last one was pretty interesting after all!"
+        else:
+           m 2m "I know that last one wasn't great...but I'm sure this next one will be better."
+    else:
+        m 1a "Say [player], would you like to hear a fun fact?"
+        m 4a "I've been looking some up just to try and teach us both something new."
+        m 4j "They say you learn something new everyday, this way I'm guaranteeing it!"
+        m 2l "I found most of these online so I can't say they're {i}definitely{/i} true... "
+        $ fun_facts_start = 1
+    m 3a "Now, let's see..."
+
+#Determines if it is a bad fact, 10% chance.
+    if fun_facts_bad_chance <= 10:
+        $ renpy.jump('bad_facts_'+str(renpy.random.randint(1,4)))
```

Comment:
While this is perfectly acceptable, you should use the renpy-native `jump` command rather than the python equivalent whenever you're in renpy script. More on that [here](https://www.renpy.org/doc/html/label.html#jump-statement). 

The syntax would be `jump expression <variable containing the label as a string>`.

Yes, this would make your one-line jumps turn into 2 lines, but with some clever work:
```renpy
if bad_chance:
    $ jumpfact = "bad_facts_{0}".format(renpy.random.randint(1,4))
else:
    $ jumpfact = "fun_fact_{0}".format(renpy.random.randint(1,20))

jump expression jumpfact
```
It'll look cleaner.

Also, the number of fun/bad facts should be constants rather than literals so its easier for us to keep track of how many facts there are.
Define these in a store like:
```renpy
define mas_funfact.fun_count = 4
```